app/_locales/en/messages.json

```diff
@@ -3321,6 +3333,36 @@
   "on": {
     "message": "On"
   },
+  "onboardedMetametricsAccept": {
+    "message": "I agree"
+  },
+  "onboardedMetametricsDisagree": {
+    "message": "No thanks"
+  },
```

Comment:
I would use existing strings when available.  For example there's already entries for "I agree" and "I disagree".  Any reason those `onboardingMetametricsAgree` `onboardingMetametricsDisagree` couldn't be used?

Reply:
Jony used the strings provided by design/content -- I think we should stick to those.  I agree there's a broader discussion to be had here for the future though.